aya/src/obj/mod.rs

```diff
@@ -662,14 +747,14 @@ mod tests {
     }
 
     fn bytes_of<T>(val: &T) -> &[u8] {
-        let size = mem::size_of::<T>();
-        unsafe { slice::from_raw_parts(slice::from_ref(val).as_ptr().cast(), size) }
+        // Safety: This is for testing only
+        unsafe { crate::util::bytes_of(val) }
```

Comment:
You can remove this 2nd definition and just import the original?

Reply:
The only problem with that is it requires adding `unsafe` in 30 something places all over the tests, so I figured a "safe" wrapper here was cheaper

Reply:
I see ok